README.rst

```diff
@@ -93,3 +158,4 @@ your option) any later version.
    :target: https://codecov.io/github/coala/coala-html?branch=master
 .. _GIF: https://cloud.githubusercontent.com/assets/7397433/16225501/8162a2a4-37c4-11e6-96b8-3e37b705c7f3.gif
 .. _Live: http://coala.github.io/coala-html
+.. _usage: https://github.com/KVGarg/coala-html/tree/build_development_instructions#usage
```

Comment:
I think this link should be changed.

Reply:
To which link should I change, can you tell me ?

Reply:
@KVGarg either the final link that you expect after merging, or a relative link (if practical and possible)